tendrl/ceph_integration/sds_sync/__init__.py

```diff
@@ -122,7 +122,9 @@ def _run(self):
                     )
 
         while not self._complete.is_set():
-            gevent.sleep(10)
+            gevent.sleep(
```

Comment:
This is not the only place that should use `sync_interval`

All the out of band object .save() with ttl should use ttl value by adding 10-15 seconds to `sync_interval`